src/recursionAssignment/Recursion.java

```diff
@@ -1,5 +1,83 @@
 package recursionAssignment;
+import java.io.*;
 
-public class Recursion{
+/**
+ * class that stores all recursion methods
+ */
 
-}
+public class Recursion{
+    /**
+     * method to count the number of instances a 7 appears in an integer
```

Comment:
use proper javadoc commenting practices and include @param and @return statement to document parameters and return values respectively for your methods